actor/echo-messaging/README.md

```diff
@@ -0,0 +1,15 @@
+# {{project-name}}
+
+This actor echoes messages received in an HttpRequest.
+It is linked to the http server provider wasmcloud:httpserver.
+
+For each http request, the actor returns a json-formatted
+string containing fields from the request.
```

Comment:
This looks like leftovers from another template,  you may want to just update this with a little info on the actor